crypto/s2n_certificate.h

```diff
@@ -53,7 +53,7 @@ typedef enum {
     S2N_CERT_TYPE_ECDSA_FIXED_ECDH = 66,
 } s2n_cert_type;
 
-struct s2n_cert_public_key {
```

Comment:
This struct currently only contains a Cert's Type and Public Key, were we planning on adding the full Cert details to it later?